srcpkgs/awesome/template

```diff
@@ -11,17 +11,40 @@ makedepends="libxcb-devel pango-devel xcb-util-devel xcb-util-image-devel
  xcb-util-keysyms-devel xcb-util-wm-devel xcb-util-cursor-devel
  startup-notification-devel imlib2-devel lua53-lgi libxdg-basedir-devel
  gdk-pixbuf-devel lua53-devel dbus-devel libxkbcommon-devel xcb-util-xrm-devel"
-depends="dbus-x11 lua53-lgi>=0.7.2 pango"
+depends="dbus-x11 $(vopt_if jit lua51-lgi lua53-lgi)>=0.7.2 pango"
```

Comment:
You're using `vopt_if` here.

Reply:
`awesome` that built with `LuaJIT` need `lua51-lgi` to work, can you tell me what's wrong with using `vopt_if` here?